bin/runtests.sh

```diff
@@ -141,6 +141,15 @@ for BUNDLE in $BUNDLES; do
     if [ $? -ne 0 ]; then
         echo $BUNDLE_NAME >> /tmp/failed.tests
     fi
+
+    comment "Restart jackrabbit"
+
+    PID=`ps -ef | grep "java -jar" | grep -v grep | awk '{ print $2 }'`
```

Comment:
Can you maybe add `jackrabbit` to the grep here? Otherwise we might stop the wrong process, in case we have more java dependencies one day.